packages/compass/src/main/handle-uncaught-exception.ts

```diff
@@ -1,8 +1,6 @@
 import { handleError } from './handle-error';
 
 async function handleUncaughtException(err: Error): Promise<void> {
-  // eslint-disable-next-line no-console
-  console.error('handling uncaughtException', err);
```

Comment:
Drive-by, `handleError` already prints the message through console.error with a better message